tools/cli/go-whisk/whisk/trigger.go

```diff
@@ -46,6 +46,20 @@ type TriggerListOptions struct {
     Docs            bool            `url:"docs,omitempty"`
 }
 
+type TriggerArray []Trigger
```

Comment:
I think you might be able to create a functions that expect an generic `interface {}` type. Then cast to the appropriate type for `Name` comparison. This way there isn't specific sort functions need for for triggers, actions, rules, and packages. See https://golang.org/doc/effective_go.html#type_switch.